R/pkg/R/schema.R

```diff
@@ -23,7 +23,8 @@
 #' Create a structType object that contains the metadata for a SparkDataFrame. Intended for
 #' use with createDataFrame and toDF.
 #'
-#' @param x a structField object (created with the field() function)
+#' @param x a structField object (created with the field() function). Since Spark 2.3, the
```

Comment:
ah, I guess we've found another pre-existing bug here-  I think this should say `\code{structField} method` instead of `field() function`